package.json

```diff
@@ -13,14 +13,18 @@
     "example": "examples"
   },
   "scripts": {
-    "test": "echo 'lol'",
-    "start": "rackt server"
+    "test": "npm run lint && echo 'lol'",
+    "start": "rackt server",
+    "lint": "eslint ."
   },
   "authors": [
     "Ryan Florence <rpflorence@gmail.com>"
   ],
   "license": "MIT",
   "devDependencies": {
+    "babel-eslint": "^4.1.3",
```

Comment:
I'd prefer these were pinned explicitly - aka `4.1.3`, I've had a few issues even with devDeps where things break and it's a bit nebulous to tell why.